packages/functional-tests/pages/resetPassword.ts

```diff
@@ -71,11 +71,11 @@ export class ResetPasswordPage extends BaseLayout {
   }
 
   get passwordResetSuccessMessage() {
-    return this.page.getByText('Your password has been reset');
+    return this.page.getByText(/Your password has been reset/);
```

Comment:
Accepting a looser match here because the success message includes a period when shown in a banner but not in alertBar